open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-api/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/ffdc/AnalyticsModelingErrorCode.java

```diff
@@ -164,7 +164,16 @@
 			"User {0} failed to create server capability {1}. Cause: {2}",
 			"The system is unable to process the request.",
 			"Review the exception to resolve the issue."),
-	
+	FAILED_UPDATE_ARTIFACT(
+			"OMAS-ANALYTICS-MODELING-027", 
+			"User {0} failed to update artifact {1}. Cause: {2}",
+			"The system is unable to process the request.",
+			"Review the exception to resolve the issue and repeat request to fix artifact."),
+	ILLEGAL_OPERATION(
+			"OMAS-ANALYTICS-MODELING-028", 
+			"Illegal operation {0} is called for instance {1}",
+			"Logical error in execution. The system is unable to process the request.",
+			"Report the error to development team."),
```

Comment:
Who is the development team?  If you mean the Egeria community then request raise an issue and describe what diagnostics are neede to fix bug

Reply:
I have simplified check if AnalyticsAsset requires update. The isModified(...) function was removed with error code in question. It was replaced by equals(...).

Reply:
This explanation needs to go into the message description